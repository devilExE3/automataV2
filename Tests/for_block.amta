for $i $:range(5)
    $:print($i)
rfo
$:print("\n")
$array = {}
$array:0 = "Hello, "
$array:1 = "World!"
$array:length = 2
for $x $array
    $:print($x)
rfo