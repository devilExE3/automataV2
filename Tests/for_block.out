01234
Hello, World!